app/swagger/swagger/requests/claim_letters.rb

```diff
@@ -0,0 +1,81 @@
+# app/controllers/v0/claim_letters_controller.rb
+module Swagger
+  module Requests
+    class ClaimLetters < ApplicationController
+      include Swagger::Blocks
+
+      swagger_path '/claim_letters' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::InternalServerError
+
+          key :description, 'Get a list of claim letters\' metadata'
+          key :tags, %w[claim_letters]
+          key :operationId, 'getClaimLetters'
+
+          response 200 do
+            key :description, 'Claim Letters metadata list'
+
+            schema do
+              key :type, :array
+
+              items do
+                key :'$ref', :ClaimLetter
+              end
+            end
+          end
+        end
+      end
+
+      swagger_path '/claim_letters/{document_id}' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::InternalServerError
+
+          key :tags, %w[claim_letters]
+          key :summary, 'Download a single PDF claim letter'
+          key :operationId, 'downloadPDFClaimLetter'
+          key :produces, ['application/pdf']
+
+          parameter do
+            key :in, :path
+            key :name, :document_id
+            key :pattern, '^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$'
+            key :description, 'Specifies the document_id (a UUID) of the claim letter to retrieve. Format: "AAAAAAAA-BBBB-CCCC-DDDD-EEEEEEEEEEEE"'
+            key :required, true
+            key :type, :string
+          end
+
+          response 200 do
+            key :description, 'Single Claim Letter PDF'
+            schema do
+              key :type, :file
+            end
+          end
```

Comment:
What other responses could we include? Does anything other than a 500 occur if there's a problem with the upstream dependency?

Reply:
Not that I know of, the only error we've seen so far in staging is 500

Reply:
If your curious about the errors connect_vbms throws: https://github.com/department-of-veterans-affairs/connect_vbms/blob/master/lib/vbms/errors.rb

I think we only need to worry about 500 for our purposes